ext/spl/spl_iterators.h

```diff
@@ -28,6 +29,7 @@
 #define spl_ce_Serializable  zend_ce_serializable
 #define spl_ce_Countable     zend_ce_countable
 #define spl_ce_Stringable    zend_ce_stringable
+#define spl_ce_JsonSerializable    php_json_serializable_ce
```

Comment:
As far as I can tell, this is never used. Maybe through macro magic?

Reply:
Yes, it's the `REGISTER_SPL_IMPLEMENTS(CachedIterable, JsonSerializable);` magic.